networks/rpc/websocket.go

```diff
@@ -96,14 +106,22 @@ func (srv *Server) FastWebsocketHandler(ctx *fasthttp.RequestCtx) {
 	}
 
 	err := upgrader.Upgrade(ctx, func(conn *fastws.Conn) {
+		if atomic.LoadInt32(&srv.wsConnCount) >= MaxWebsocketConnections {
+			return
+		}
+		atomic.AddInt32(&srv.wsConnCount, 1)
+		wsConnCounter.Inc(1)
+		defer func() {
+			atomic.AddInt32(&srv.wsConnCount, -1)
+			wsConnCounter.Dec(1)
+		}()
 		//Create a custom encode/decode pair to enforce payload size and number encoding
 		encoder := func(v interface{}) error {
 			msg, err := json.Marshal(v)
 			if err != nil {
 				return err
 			}
 			err = conn.WriteMessage(websocket.TextFrame, msg)
-			conn.Close()
```

Comment:
@jeongkyun-oh Do you know why this line was inserted?

Reply:
Not really, I guess it was for testing purpose. I checked that the `hijackConn` return nil.